### ISSUE TITLE
Cifar100 Finetune NAN

### ISSUE THREAD

**Issue by gaow0007** [Distributed-AI/PipeTransformer#5]
When I reproduce cifar100 finetuning result on ViT. I notice that you comment [this line](https://github.com/Distributed-AI/PipeTransformer/blob/master/examples/image_classification/main_cv.py#L69). 

I comment / uncomment [this line](https://github.com/Distributed-AI/PipeTransformer/blob/master/examples/image_classification/main_cv.py#L69). Both lead to NAN loss within 1 epoch. 

May I ask you to provide your experimental log of cifar100 finetuning.

**Comment by chaoyanghe** [comment]
> When I reproduce cifar100 finetuning result on ViT. I notice that you comment [this line](https://github.com/Distributed-AI/PipeTransformer/blob/master/examples/image_classification/main_cv.py#L69).
> 
> I comment / uncomment [this line](https://github.com/Distributed-AI/PipeTransformer/blob/master/examples/image_classification/main_cv.py#L69). Both lead to NAN loss within 1 epoch.
> 
> May I ask you to provide your experimental log of cifar100 finetuning.

Sure. I found a log here:
[cifar100-pipetransformer.txt](https://github.com/Distributed-AI/PipeTransformer/files/8684698/cifar100-pipetransformer.txt)

**Comment by chaoyanghe** [comment]
```
 ____   _               _____                            __                                      _ 
|  _ \ (_) _ __    ___ |_   _| _ __   __ _  _ __   ___  / _|  ___   _ __  _ __ ___    ___  _ __ | |
| |_) || || '_ \  / _ \  | |  | '__| / _` || '_ \ / __|| |_  / _ \ | '__|| '_ ` _ \  / _ \| '__|| |
|  __/ | || |_) ||  __/  | |  | |   | (_| || | | |\__ \|  _|| (_) || |   | | | | | ||  __/| |   |_|
|_|    |_|| .__/  \___|  |_|  |_|    \__,_||_| |_||___/|_|   \___/ |_|   |_| |_| |_| \___||_|   (_)
          |_|                                                                                      

90550 2021-02-03,01:58:52.358 - {auto_dp.py (322)} - _build_broad_cast_message(): 
################################ Congratulations! To train faster, PipeTransformer has automatically transformed to:
################################ Epoch: 2 
################################ Number of frozen layers: 7 
################################ Pipe length: 4/8 
################################ Newly added ranks: [1, 9] 
```

**Comment by chaoyanghe** [comment]
> 

When transformation happens, there would be log like this one. You can check around about the loss

**Comment by gaow0007** [comment]
Thanks for your reply.
